### PR TITLE
S14 product suggested

### PR DESCRIPTION
**Pull request by lingsat**

### CODE REVIEW

File: src/features/products/components/Card/Card.tsx [hunk 1]
```diff
@@ -26,12 +27,12 @@ const Card: FC<CardProps> = ({ product }) => {
...
 
   const handleOpenProduct = () => {
-    navigate(`/products/${product.id}`);
+    navigate(`/${ERoutes.PRODUCTS_LIST}/${product.id}`);
   };
 
   return (
     <li className="card">
-      <Link to={`/products/${product.id}`}>
+      <Link to={`/${ERoutes.PRODUCTS_LIST}/${product.id}`}>
```
Comment (Yuliia143) [inline src/features/products/components/Card/Card.tsx]: Remove duplicates

File: src/features/products/components/Card/Card.tsx [hunk 2]
```diff
@@ -41,7 +42,9 @@ const Card: FC<CardProps> = ({ product }) => {
       <div className="card__info">
         <div className="info__left">
           <h3 className="info__title">
-            <Link to={`/products/${product.id}`}>{product.title}</Link>
+            <Link to={`/${ERoutes.PRODUCTS_LIST}/${product.id}`}>
```
Comment (Yuliia143) [inline src/features/products/components/Card/Card.tsx]: x2

**Review by Yuliia143 — CHANGES_REQUESTED** [review]

File: src/features/products/components/Card/Card.tsx (hunk 1 above)
Comment (lingsat) [reply src/features/products/components/Card/Card.tsx]: Remove Links. Use onClick navigate for img, titile, button

File: src/features/products/components/Card/Card.tsx (hunk 2 above)
Comment (lingsat) [reply src/features/products/components/Card/Card.tsx]: remove links

**Review by Yuliia143 — APPROVED** [review]

**Review by Yuliia143 — APPROVED** [review]

**Pull request merged**